app/controllers/signs_controller.rb

```diff
@@ -8,13 +8,19 @@ class SignsController < ApplicationController
   before_action :set_sign, only: [:remove_slide, :poll, :show, :edit, :update, :destroy, :play, :settings, :order]
   before_action :set_search_filters, only: [:index]
 
-  load_and_authorize_resource except: [:play, :poll]
+  load_and_authorize_resource except: :poll
```

Comment:
Are we sure this will not cause problems? How do campus monitors access play screen?